packages/design-system/src/components/Alert/Alert.test.tsx

```diff
@@ -41,8 +53,8 @@ describe('Alert', function () {
   it('renders additional className and role prop', () => {
     const className = 'ds-u-test';
     const role = 'alert';
-    renderAlert({ className, role });
-    const alert = screen.getByRole(role);
+    renderAlert({ 'class-name': className, role });
+    const alert = document.querySelector('.ds-c-alert');
```

Comment:
had to change the `getByRole` query because we're adding a native HTML attr (`role`) to the web component itself; `role` appears on both the root element and internally so this test couldn't find the right element to test